apps/dashboard/src/components/container/ContainersCard.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <CardComponent :header="t('containersOverview.Containers')" class="p-0">
+  <CardComponent :header="t('modules.seller.productContainers.containers.header')" class="p-0">
     <template #topAction>
       <div v-if="showCreate || associatedPos" class="flex justify-content-endg gap-2">
         <Button
             v-if="associatedPos"
             @click="openContainerAdd()"
-            outlined>{{ t('containersOverview.addExisting')}}</Button>
-        <Button @click="openContainerEdit()">{{t('app.Create')}}</Button>
+            outlined>{{ t('modules.seller.productContainers.containers.addExisting') }}</Button>
+        <Button @click="openContainerEdit()">{{t('common.create')}}</Button>
```

Comment:
Maybe add something along the lines of [vue/mustache-interpolation-spacing](https://eslint.vuejs.org/rules/mustache-interpolation-spacing) to enforce consistent spacing for template syntax (most of the times you use single spacing around, in some rare cases you forget it).